src/Styleguide/Components/Artwork/ImageCarousel.tsx

```diff
@@ -24,6 +24,14 @@ const BaseImageArea = styled.div`
 
 const SmallImageArea = styled(BaseImageArea)`
   max-height: calc(100vh - 120px);
+
+  /**
+   * This will make sure the image area already exists prior to image loading
+   * and the layout won’t jump later on, but it does rely on the current design
+   * where this image is supposed to be shown from the left side of the screen
+   * to the right side of the screen, i.e. the width of the viewport.
+   */
+  height: calc(100vw / ${({ aspectRatio }) => aspectRatio});
```

Comment:
This all looks good to me. Just trying to understand - here we are trying to make the image height proportional to view width? This will guarantee that really wide images do not take the whole width of the screen? That is an idea?

But should not the height of the image be the same for all the images in the carousel?  Or the idea is that it will jump in size when going image to image?

Reply:
On mobile, the image carousel does not have a height relative to the vertical viewport, like it does on desktop. So when the page initially loads, the image area has a 0 height, then once the browser has downloaded the image the page jumps as the image area grows to fit the image. This bit of CSS immediately sets the height on the initial page load, because we know the aspect ratio and the width.